templates-java/src/main/java/br/com/templates_java/config/jms_aws_sqs/ProducerMessageGenerator.java

```diff
@@ -0,0 +1,24 @@
+package br.com.templates_java.config.jms_aws_sqs;
+
+
+import br.com.generator.core.Generator;
+import br.com.generator.core.GeneratorOptions;
+
+import java.io.File;
+import java.io.IOException;
+
+public class ProducerMessageGenerator extends Generator {
+
+    private GeneratorOptions generatorOptions;
+
+    public ProducerMessageGenerator(GeneratorOptions generatorOptions) {
```

Comment:
The constructor parameter `generatorOptions` could be checked for null to prevent potential `NullPointerException`